nav2_util/include/nav2_util/geometry_utils.hpp

```diff
@@ -144,16 +144,17 @@ inline Iter min_by(Iter begin, Iter end, Getter getCompareVal)
  * Find first element in iterator that is greater integrated distance than comparevalue
  */
 template<typename Iter, typename Getter>
-inline Iter first_after_integrated_distance(Iter begin, Iter end, Getter getCompareVal)
+inline Iter first_after_integrated_distance(Iter begin, Iter end, Getter getCompareVal, bool backward=false)
```

Comment:
I believe by the nature of this function being templated on iterators, you could instead give it reverser iterators and require no changes.

Reply:
Right, I updated the commit accordingly.